src/vtlengine/Operators/Numeric.py

```diff
@@ -390,3 +381,37 @@ def py_op(cls, x: float, param: Optional[float]) -> Any:
             return truncated_value
 
         return int(truncated_value)
+
+
+class PseudoRandom(_random.Random):
+
+    def __init__(self, seed: int) -> None:
+        super().__init__()
+        self.seed(seed)
+
+
+class Random(Parameterized):
+
+    op = RANDOM
+    return_type = Number
+
+    @classmethod
+    def validate(cls, seed: Any, index: Any = None) -> Any:
+        if index.data_type != Integer:
+            index.data_type = binary_implicit_promotion(index.data_type, Integer)
+        if index.value < 0:
+            raise SemanticError("2-1-15-2", op=cls.op, value=index)
+        if index.value > 10000:
+            warnings.warn("The value of 'index' is very big. This can affect "
```

Comment:
Mention the random operator in the warning message

Reply:
Now it says: "Random: The value of 'index' is very big. This can affect performance."